app/code/Magento/Sales/view/frontend/templates/order/print/creditmemo.phtml

```diff
@@ -34,11 +34,10 @@
                 </tr>
             </thead>
             <?php $_items = $_creditmemo->getAllItems(); ?>
-            <?php $_count = count($_items); ?>
             <?php foreach ($_items as $_item): ?>
-            <?php if ($_item->getOrderItem()->getParentItem()) {
-    continue;
-} ?>
+            <?php if ($_item->getOrderItem()->getParentItem()): ?>
+                continue;
```

Comment:
```continue;``` will be shown. It is regression. Am I wrong?

Reply:
@ihor-sviziev @orlangur thank you for your review. Yep, it was an error. Thanks once more for your help!